work_test.go

```diff
@@ -238,21 +239,19 @@ func TestLockJobAdvisoryRace(t *testing.T) {
 	getBackendID := func(conn *pgx.Conn) int32 {
 		var backendID int32
 		err := conn.QueryRow(`
-			SELECT backendid
-			FROM pg_stat_get_backend_idset() psgb(backendid)
-			WHERE pg_stat_get_backend_pid(psgb.backendid) = pg_backend_pid()
+			SELECT pg_backend_pid()
 		`).Scan(&backendID)
```

Comment:
The value here is changing from the "backend ID" used by the [per-backend stats functions](https://www.postgresql.org/docs/9.6/static/monitoring-stats.html#MONITORING-STATS-BACKEND-FUNCS-TABLE) to the "process ID." I think the names should change accordingly. Perhaps `backendPID` here and elsewhere?